maplibre_gl_platform_interface/lib/src/method_channel_maplibre_gl.dart

```diff
@@ -255,6 +255,104 @@ class MapLibreMethodChannel extends MapLibrePlatform {
     return await _channel.invokeMethod('map#getTelemetryEnabled');
   }
 
+  @override
+  Future<void> setMaximumFps(int fps) async {
+    await _channel.invokeMethod('map#setMaximumFps', <String, dynamic>{
+      'fps': fps,
+    });
+  }
+
+  @override
+  Future<void> forceOnlineMode() async {
+    await _channel.invokeMethod('map#forceOnlineMode');
+  }
+
+  @override
+  Future<bool> easeCamera(CameraUpdate cameraUpdate, int? duration) async {
+    final result = await _channel.invokeMethod('camera#ease', <String, dynamic>{
+      'cameraUpdate': cameraUpdate.toJson(),
+      'duration': duration,
+    });
+    return result as bool;
+  }
```

Comment:
Expose `Duration` class for the `duration` argument. Then, use `Duration.inMilliseconds` getter to pass milliseconds to the `MethodChannel` method args.
Like `animateCamera`, the duration argument must be a named parameter, so:

```suggestion
  Future<bool> easeCamera(CameraUpdate cameraUpdate, {Duration? duration}) async {
    final result = await _channel.invokeMethod('camera#ease', <String, dynamic>{
      'cameraUpdate': cameraUpdate.toJson(),
      'duration': duration?.inMilliseconds,
    });
    return result as bool;
  }
```